mappings/net/minecraft/util/registry/DynamicRegistryManager.mapping

```diff
@@ -25,26 +29,83 @@ CLASS net/minecraft/unmapped/C_wqxmvzdq net/minecraft/util/registry/DynamicRegis
+		ARG 0 key
+	METHOD m_fpwtdfne ownedNetworkableRegistries ()Ljava/util/stream/Stream;
+	METHOD m_jffdyfoo retreiveRegistry (Lnet/minecraft/unmapped/C_xhhleach;Lcom/mojang/serialization/Dynamic;)Lnet/minecraft/unmapped/C_tqxyjqsk;
+		ARG 0 key
 	METHOD m_kagwftqy getMutable (Lnet/minecraft/unmapped/C_xhhleach;)Lnet/minecraft/unmapped/C_tqxyjqsk;
 		ARG 1 key
 	METHOD m_lbyadhxv load (Lnet/minecraft/unmapped/C_wqxmvzdq$C_qndozwiw;Lcom/mojang/serialization/DynamicOps;Lnet/minecraft/unmapped/C_yftyvswu;)V
 		COMMENT Loads a dynamic registry manager from the resource manager's data files.
+		ARG 0 writable
+		ARG 2 loader
+	METHOD m_lheqxlni blankWritable ()Lnet/minecraft/unmapped/C_wqxmvzdq$C_qndozwiw;
+	METHOD m_lxobzcah fromRegistryOfRegistries (Lnet/minecraft/unmapped/C_tqxyjqsk;)Lnet/minecraft/unmapped/C_wqxmvzdq$C_ggmzysrd;
+		ARG 0 key
+	METHOD m_matjkzip captureMap (Lcom/mojang/serialization/codecs/UnboundedMapCodec;)Lcom/mojang/serialization/Codec;
 	METHOD m_nnjsvctt load (Lcom/mojang/serialization/DynamicOps;Lnet/minecraft/unmapped/C_yftyvswu$C_ogvxrsre;Lnet/minecraft/unmapped/C_wqxmvzdq$C_ydhbsuwm;)V
 		COMMENT Loads elements from the {@code ops} into the registry specified by {@code
 		COMMENT info} within the {@code manager}. Note that the resource manager instance
 		COMMENT is kept within the {@code ops}.
+		ARG 1 bound
 		ARG 2 info
-	METHOD m_physpezp (Lnet/minecraft/unmapped/C_wqxmvzdq$C_qndozwiw;Lnet/minecraft/unmapped/C_nyvkclsy$C_dxgwghhk;Lnet/minecraft/unmapped/C_wqxmvzdq$C_ydhbsuwm;)V
+	METHOD m_physpezp addBuildInElements (Lnet/minecraft/unmapped/C_wqxmvzdq$C_qndozwiw;Lnet/minecraft/unmapped/C_nyvkclsy$C_dxgwghhk;Lnet/minecraft/unmapped/C_wqxmvzdq$C_ydhbsuwm;)V
+		ARG 0 writable
+		ARG 1 inMemoryStorage
 		ARG 2 info
+	METHOD m_pkyudotx networkSafeRegistries ()Ljava/util/stream/Stream;
 	METHOD m_pquswexb getOptionalMutable (Lnet/minecraft/unmapped/C_xhhleach;)Ljava/util/Optional;
 		COMMENT Retrieves a registry optionally from this manager.
 		ARG 1 key
+	METHOD m_qepmomyf makeNetworkCodec ()Lcom/mojang/serialization/Codec;
+	METHOD m_qfxecdvh globalRegistries ()Ljava/util/stream/Stream;
+	METHOD m_sloxobqt freeze ()Lnet/minecraft/unmapped/C_wqxmvzdq$C_ggmzysrd;
 	METHOD m_tcjkywtj builtInCopy ()Lnet/minecraft/unmapped/C_wqxmvzdq$C_qndozwiw;
+	METHOD m_uezzwhkz createRegistry (Lnet/minecraft/unmapped/C_xhhleach;)Lnet/minecraft/unmapped/C_eexxncvi;
+		ARG 0 key
+	METHOD m_vutgtdyk registries ()Ljava/util/stream/Stream;
+	METHOD m_xnavlkkh createFreshRegistries ()Ljava/util/Map;
 	METHOD m_zbdqwvdm register (Lcom/google/common/collect/ImmutableMap$Builder;Lnet/minecraft/unmapped/C_xhhleach;Lcom/mojang/serialization/Codec;Lcom/mojang/serialization/Codec;)V
 		ARG 0 infosBuilder
 		ARG 1 registryRef
 		ARG 2 entryCodec
 		ARG 3 networkEntryCodec
+	METHOD m_zygzorae readFromDisk (Lcom/mojang/serialization/Dynamic;)Lnet/minecraft/unmapped/C_wqxmvzdq;
+	CLASS C_ggmzysrd Frozen
+	CLASS C_gmaxoias ImmutableRegistryAccess
```

Comment:
Is the super class of this one a `RegistryManager`? If so, the name should be `ImmutableRegistryManager`